content/chainguard/chainguard-images/retrieve-image-sboms.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "How to Retrieve SBOMs for Chainguard Images"
+linktitle: "Retrieve an Image's SBOM"
+type: "article"
+description: "A brief tutorial on how to use Cosign to retrieve Chainguard Image SBOMs."
+date: 2023-11-17T11:07:52+02:00
+lastmod: 2023-11-17T11:07:52+02:00
+draft: false
+tags: ["Conceptual", "Chainguard Images", "SBOM"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 710
+toc: true
+---
+
+
+Chainguard Images contain only the minimum number of packages needed to use the software they contain. The purpose of this is to reduce the image's attack surface and minimize the risk that CVEs will impact software that depends on these container images. 
+
+Even though they contain the minimum number of packages, there may come a time when you want to know exactly what's running inside of a certain Chainguard Image. For this reason, we include a signed SBOM (also known as an *attestation*) with each image.
+
+[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/) — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/what-is-the-oci/) registry, as well as the retrieval of said artifacts. This tutorial outlines how you can use the `cosign` command to retrieve a Chainguard Image's SBOM using Cosign. 
+
+
+## Prerequisites
+
+In order to follow this guide, you'll need the following installed on your local machine:
+
+* **Cosign** — to retrieve SBOMs associated with Chainguard Images, check out [our guide on installing Cosign](/open-source/sigstore/cosign/how-to-install-cosign/) to configure it.
+* **jq** — to process JSON, visit the [jq downloads](https://jqlang.github.io/jq/download/) page to set it up.
+
+
+## Using Cosign to retrieve an Image's SBOM
+
+Cosign includes a `download` command that allows you to retrieve a Chainguard Image's SBOM over the command line. To do so, you would use this command with syntax like the following.
+
+```shell
+cosign download attestation \ 
+cgr.dev/chainguard/$IMAGE | jq -r .payload | base64 -d | jq .predicate
+```
+
+Notice that this example syntax includes `download attestation` rather than `download sbom`. While there are a few [differences between SBOMs and attestations](/open-source/sbom/sboms-and-attestations/), you can generally think of an attestation as a more accurate representation of the contents of the container image than an SBOM. This is because attestations must be signed by the software producer, thereby ensuring the accuracy of the SBOM and the quality of the software. 
```

Comment:
```suggestion
Notice that this example syntax includes `download attestation` rather than `download sbom`. You can generally think of an attestation as a an authenticated statement about a software artifact. There are different types of attestations [as defined by the SLSA 1.0 specification](https://slsa.dev/attestation-model), and they are typically referenced by their **predicate type**. One of the available predicate types is SPDX, an open standard for SBOM files. Because attestations must be signed, this is a form of verifying the  authenticity of the software producer, thereby ensuring the accuracy of the SBOM and the quality of the software. 
```

Reply:
the attestation is in fact like an envelope that wraps the SBOM with additional metadata and is signed, so it can be verified. I hope this helps clarifying it a bit more, but feel free to move things around for better fluidity :)

Reply:
This is really helpful framing, thanks Erika!